models/labels/addresses/__single_category_labels__/likely_bots/optimism/labels_optimism_likely_bot_addresses.sql

```diff
@@ -21,10 +21,10 @@ WITH sender_transfer_rates AS (
             , SUM(CASE WHEN to IN (SELECT address FROM {{ ref('labels_optimism_likely_bot_contracts') }} WHERE name != 'chain ops bot') THEN 1 ELSE 0 END) AS bot_concentration_txs
             
             , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ source('erc20_optimism','evt_Transfer') }} r WHERE t.hash = r.evt_tx_hash AND t.block_number = r.evt_block_number) THEN 1 ELSE 0 END) AS num_erc20_tfer_txs
-            , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ ref('nft_transfers') }} r WHERE t.hash = r.tx_hash AND t.block_number = r.block_number AND blockchain = 'optimism') THEN 1 ELSE 0 END) AS num_nft_tfer_txs
+            , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ ref('nft_transfers_legacy') }} r WHERE t.hash = r.tx_hash AND t.block_number = r.block_number AND blockchain = 'optimism') THEN 1 ELSE 0 END) AS num_nft_tfer_txs
```

Comment:
I am still confused with when to use `ref('\*_legacy')`, in legacy models or in dune sql models.

Reply:
just clarifying that we spoke in dm's about this, resolving now